tools/setup-envtest/env/env.go

```diff
@@ -26,24 +26,28 @@ import (
 // envtest binaries.
 //
 // In general, the methods will use the Exit{,Cause} functions from this package
-// to indicate errors.  Catch them with a `defer HandleExitWithCode()`.
+// to indicate errors. Catch them with a `defer HandleExitWithCode()`.
 type Env struct {
 	// the following *must* be set on input
 
 	// Platform is our current platform
 	Platform versions.PlatformItem
 
-	// VerifiySum indicates whether or not we should run checksums.
+	// VerifySum indicates whether we should run checksums.
 	VerifySum bool
-	// NoDownload forces us to not contact GCS, looking only
-	// at local files instead.
+	// NoDownload forces us to not contact GCS or download the index via HTTP,
+	// looking only at local files instead.
 	NoDownload bool
 	// ForceDownload forces us to ignore local files and always
-	// contact GCS & re-download.
+	// contact GCS or download the index via HTTP & re-download.
 	ForceDownload bool
 
-	// Client is our remote client for contacting GCS.
-	Client *remote.Client
+	// UseGCS signals if the GCS client is used.
+	UseGCS bool
```

Comment:
```suggestion
	// UseLegacyGCSBucket signals if the GCS client is used.
	UseLegacyGCSBucket bool
```

Reply:
I'll use UseDeprecatedGCS instead. I would use the same in the command line flag and this will make it a lot clearer that it goes away